documentation/docs/practical-sessions/configuring-station-metadata.md

```diff
@@ -4,96 +4,132 @@ title: Configuring station metadata
+Output:
+    
+```{.copy}
+Continue with token: MyCatIsCalledJessie! [y/N]? y
+Token successfully created
 ```
 
-### Review your station list
+!!! note "Exercise 1: Create an authorization token for collections/stations"
 
-Check the content of your station list from the command line as follows:
+    Please create an authorization token for the `collections/stations` endpoint using the instructions above.
 
-```bash
-cat ~/wis2box-data/metadata/station/station_list.csv
-```
+## add station metadata using the wis2box-webapp
 
-Or open the file in WinSCP.
+The wis2box-webapp provides a graphical user interface to edit station metadata.
 
-Keep adding lines to `station_list.csv` and ensure you have at least three stations defined.
+Open the wis2box-webapp in your browser: `http://<your-host>/wis2box-app`:
 
-## Publishing station metadata
+<img alt="wis2box-webapp" src="../../assets/img/wis2box-webapp.png" width="800">
 
-Login in to the **wis2box-management** container:
+And select stations:
 
-```bash
-cd ~/wis2box-1.0b4/
-python3 wis2box-ctl.py login
-```
+<img alt="wis2box-webapp-select-stations" src="../../assets/img/wis2box-webapp-select-stations.png" width="250">
 
-Run the following command to publish your station metadata:
+When you click add 'add new station' you are asked to provide the WIGOS-station-identifier for the station you want to add:
 
-```bash
-wis2box metadata station publish-collection
-```
+<img alt="wis2box-webapp-import-station-from-oscar" src="../../assets/img/wis2box-webapp-import-station-from-oscar.png" width="800">
+
+When you click search the station data is retrieved from OSCAR, please note that this can take a few seconds.
+
+Review the data returned by OSCAR and add missing data where required. Select a topic for the station and provide your authorization token for the `collections/stations` endpoint and click 'save' and the station will be saved:
+
+<img alt="wis2box-webapp-create-station-save" src="../../assets/img/wis2box-webapp-create-station-save.png" width="800">
+
+<img alt="wis2box-webapp-create-station-success" src="../../assets/img/wis2box-webapp-create-station-success.png" width="500">
+
+Go back to the station list and you will see the station you added:
+
+<img alt="wis2box-webapp-stations-with-one-station" src="../../assets/img/wis2box-webapp-stations-with-one-station.png" width="800">
+
+!!! note "Exercise 2: Add station metadata"
+
+    Please add three or more stations to the wis2box station metadata collection of your wis2box. 
+    
+    Please use stations from your country if possible, especially if you brought your own data.
+    
+    Otherwise, you can use the following WIGOS-station-identifiers for testing purposes:
+
+    - 0-20000-0-91334
+    - 0-20000-0-96323 (note missing station elevation in OSCAR)
+    - 0-20000-0-96749 (note missing station elevation in OSCAR)
+
```

Comment:
Add a hint / note that if the elevation is missing there are web services / API that allow you to look up the elevation using open data. One such example is the opentopodata API, e.g. to get the elevvation of the BMKG auditorium we would use https://api.opentopodata.org/v1/aster30m?locations=-6.15558,106.84204